packages/binaryninja.vm/binaryninja.vm.nuspec

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>binaryninja.vm</id>
+    <version>4.2.6455.20250417</version>
+    <authors>Vector 35, Inc.</authors>
+    <description>Binary Ninja is an interactive decompiler, disassembler, debugger, and binary analysis platform built by reverse engineers, for reverse engineers.</description>
```

Make the description as short as possible to follow [our documentation](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#description). This also impacts https://github.com/mandiant/VM-Packages/issues/1328 and https://github.com/mandiant/VM-Packages/issues/1231

```suggestion
    <description>Binary Ninja is an interactive decompiler, disassembler, debugger, and binary analysis platform.</description>
```

---

Done.